Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudtruth"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
We generally don't bump this until the release is ready to go, since we don't know a priori what the next release number will be. We can give it a suffix, like "-dev", "-pre", or "-snapshot" if you'd like.

Reply:
Got it, I'll revise that to 0.1.3-pre or something.